http/handler.go

```diff
@@ -280,6 +280,11 @@ func wrapGenericHandler(core *vault.Core, h http.Handler, props *vault.HandlerPr
 	if maxRequestSize == 0 {
 		maxRequestSize = DefaultMaxRequestSize
 	}
+
+	// Swallow this error since we don't want to pollute the logs and we also don't want to
+	// return an HTTP error here. This information is best effort.
+	hostname, _ := os.Hostname()
```

Comment:
Am I correct in assuming that moving this assignment outside of the handler function we return means it's only called once, rather than on every API request?

Reply:
That's my read of it too.  You could always verify with a log statement if you want to be sure.